internal/cmd/policy/listrules/listrules.go

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package listrules
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/policy/persistent"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p *persistent.Options
+}
+
+func (o *options) AddFlags(_ *cobra.Command) {}
+
+func (o *options) Run(cmd *cobra.Command, _ []string) error {
+	repo, err := repository.LoadRepository()
+	if err != nil {
+		return err
+	}
+
+	rules, err := repo.ListRules(cmd.Context())
+	if err != nil {
+		return err
+	}
+
+	for _, rule := range rules {
+		if rule.Name == "gittuf-allow-rule" {
+			continue
+		}
+
+		fmt.Printf("Rule %s:\n", rule.Name)
+		m := map[string][]string{}
+		for _, path := range rule.Paths {
+			if strings.HasPrefix(path, "git:") {
+				m["git"] = append(m["git"], path)
```

Comment:
The slices haven't been initialized right? I expect a panic here. FWIW I don't think the map is necessary. You could just have two initialized slices for git and file rules and append to them.

Reply:
Fixed